packages/contracts/deploy-config/local.ts

```diff
@@ -12,6 +12,7 @@ const config = {
   bvmFeeWalletAddress: process.env.BVM_FEE_WALLET_ADDRESS || '0x391716d440c151c42cdf1c95c1d83a5427bca52c',
   bvmAddressManagerOwner: process.env.BVM_ADDRESS_MANAGER_OWNER || '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
   bvmGasPriceOracleOwner: process.env.BVM_GAS_PRICE_ORACLE_OWNER || '0x9965507d1a55bcc2695c58ba16fb37d819b0a4dc',
+  l1BitAddress: process.env.L1_BIT_ADDRESS || '0x89F06180e62a6d3e5ac130bbCE7bD004b434100b',
```

Comment:
default L1_BIT_ADDRESS is not zeroAddress?

Reply:
It must be L1 bit erc20 address